src/System.Runtime/tests/System/Reflection/RuntimeReflectionExtensionsTests.cs

```diff
@@ -42,24 +49,67 @@ public void GetRuntimeEvent()
             Assert.Equal(typeof(TestType).GetEvent("StuffHappened"), typeof(TestType).GetRuntimeEvent("StuffHappened"));
         }
 
+        [Fact]
+        public void GetRuntimeEventOnNull() => Assert.Throws<ArgumentNullException>("type", () => default(Type).GetRuntimeEvent(""));
```

Comment:
For Assert.Throws that validate argument names, please use the AssertExtensions.Throws equivalent.  That let's us ignore the name validation on check on platforms like .NET Native that get rid of unneeded metadata in release builds.

Reply:
Oops. Plum forgot about that.

Reply:
:smile: